stack.yaml.lock

```diff
@@ -0,0 +1,75 @@
+# This file was autogenerated by Stack.
```

Comment:
This file shouldn't be committed. You can do a `git reset --soft` to open the commit back up, remove the change, and then force-push.

Reply:
Ah nevermind, I see you specifically committed this in a single commit. Hm, then you'd have to do some rebasing.

Reply:
Stack recommends [committing the lock file](https://docs.haskellstack.org/en/stable/lock_files/) for reproducibility. Is there a reason it shouldn't be committed in this project?

Reply:
Oh, that's new information for me. In that case, let's keep it.